index.rst

```diff
@@ -25,131 +25,155 @@ What can we help you with today?
 
-    ---
+        .. button-link:: docs/products/cassandra
+            :align: right
+            :color: primary
+            :outline:
 
-    |icon-redis| **Redis®\*** In-memory data store for all your high-peformance short-term storage and caching needs.
+            Read more
+    
+    .. grid-item-card::
+        :shadow: md
+        :margin: 2 2 0 0
 
-    +++
+        |icon-redis| **Redis®\*** In-memory data store for all your high-peformance short-term storage and caching needs.
 
-    .. link-button:: docs/products/redis
-        :type: ref
-        :text: Read more
-        :classes: stretched-link
+        .. button-link:: docs/products/redis
+            :align: right
+            :color: primary
+            :outline:
 
-    ---
+            Read more
+    
 
-    |icon-mysql| **MySQL** Popular and much-loved relational database platform.
+    .. grid-item-card::
+        :shadow: md
+        :margin: 2 2 0 0
 
-    +++
+        |icon-mysql| **MySQL** Popular and much-loved relational database platform.
 
-    .. link-button:: docs/products/mysql
-        :type: ref
-        :text: Read more
-        :classes: stretched-link
+        .. button-link:: docs/products/mysql
+            :align: right
+            :color: primary
+            :outline:
 
-    ---
+            Read more
+    
+    .. grid-item-card::
+        :shadow: md
+        :margin: 2 2 0 0
 
-    |icon-influxdb| **InfluxDB®** Specialist time series database, with good tooling support.
+        |icon-influxdb| **InfluxDB®** Specialist time series database, with good tooling support.
 
-    ---
+    .. grid-item-card::
+        :shadow: md
+        :margin: 2 2 0 0
 
-    |icon-grafana| **Grafana®** The visualization tool you need to explore and understand your data. Grafana integrates with the other services in just a few clicks.
+        |icon-grafana| **Grafana®** The visualization tool you need to explore and understand your data. Grafana integrates with the other services in just a few clicks.
 
-    +++
+        .. button-link:: docs/products/grafana
+            :align: right
+            :color: primary
+            :outline:
 
-    .. link-button:: docs/products/grafana
-        :type: ref
-        :text: Read more
-        :classes: stretched-link
+            Read more
     
-    ---
-
-    |icon-clickhouse| **ClickHouse** A highly scalable, open source database that uses a column-oriented structure. :badge:`beta,cls=badge-secondary text-black badge-pill`
+    .. grid-item-card::
+        :shadow: md
+        :margin: 2 2 0 0
 
-    +++
+        |icon-clickhouse| **ClickHouse** A highly scalable, open source database that uses a column-oriented structure. :bdg-secondary:`beta`
 
-    .. link-button:: docs/products/clickhouse
-        :type: ref
-        :text: Read more
-        :classes: stretched-link
+        .. button-link:: docs/products/clickhouse
+            :align: right
+            :color: secondary
```

Comment:
Here we have "secondary", unlike other cases where color is set to "primary". In Light mode there is no differece. However, on hover in dark mode - the secondary looks way better to me! it has better contrast of the text. I'd prefer if we set secondary for others as well